integration_tests/models/expected/snowplow_unified_views_mobile_screen_engagement_expected_stg.sql

```diff
@@ -0,0 +1,29 @@
+{#
+Copyright (c) 2024-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Personal and Academic License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Personal and Academic License Version 1.0.
+You may obtain a copy of the Snowplow Personal and Academic License Version 1.0 at https://docs.snowplow.io/personal-and-academic-license-1.0/
+#}
+
+
+
+select
+    view_id,
+    session_identifier,
+    user_identifier,
+    engaged_time_in_s,
+    {% if target.type in ['bigquery'] %}
+    -- BiqQuery calculates timestamp difference slightly differently
+    coalesce(absolute_time_in_s_bigquery, absolute_time_in_s) as absolute_time_in_s,
```

Comment:
Just flagging that I had to include this hack, because the `snowplow_utils.timestamp_diff` gives different values on BigQuery than on other warehouses (off by one in some cases, so probably a result of how it handles the milliseconds). Please let me know if there is something that I am missing!

Reply:
I don't remember how we hacked around it before, but yeah we know they can be different so that's fine!